### ISSUE TITLE
False positives for parametric mixins with rule block/snippet in indentation

### ISSUE THREAD

**Issue by timbru31** [stylelint/stylelint#2271]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug where stylelint does not treat parameters in case a ruleblock/snippet is passed.

> What CSS is needed to reproduce this issue?

```css
#root {
	.foo(@name, @rules) {
		.@{name} {
			@rules();
		}
	}
}

#root > .foo(test, {
	margin-bottom: 15px;
});
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
	"rules": {
		"indentation": "tab"
	}
}

```

> Which version of stylelint are you using?

7.7.1

> How are you running stylelint: CLI, PostCSS plugin, Node API?

./node_modules/.bin/stylelint test.less

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, Less mixins.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged

```shell
test.less
 10:2  ✖  Expected indentation of 0 tabs   indentation
```

**Comment by jeddy3** [comment]
@timbru31 Thanks for the report and for using the issue template.

Supporting non-standard syntax is currently a low priority for the core team. As such, please consider investigating and, if necessary, [contributing](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md) a fix to the [`indentation`](https://github.com/stylelint/stylelint/tree/master/lib/rules/indentation) rule.

**Comment by webOS101** [comment]
I have created a patch for this that fixes all the less mixin false positives in our code base.  Be interested if it also fixes your use cases.

**Comment by jeddy3** [comment]
Done in https://github.com/stylelint/stylelint/issues/2744

**Comment by timbru31** [comment]
I can confirm the fix, thanks! 🎉
